terratest/test/k8gb_basic_app_test.go

```diff
@@ -50,9 +50,9 @@ func TestK8gbBasicAppExample(t *testing.T) {
 	require.Equal(t, ingress.Name, "test-gslb")
 
 	// Path to the Kubernetes resource config we will test
-	unhealthyAppPath, err := filepath.Abs("../examples/unhealthy-app.yaml")
+	not-foundAppPath, err := filepath.Abs("../examples/not-found-app.yaml")
```

Comment:
hmm, this naming doesn't look good

Reply:
yeah, also noticed it after failing terratest. Some evil side effect of mass renaming. Fix is in test now